### PR TITLE
ASA-213: Bump minor version instead; update `cran_comments.md`

### PR DESCRIPTION
**Pull request by tyler-richardett**
### Description

Bumped minor version of library (to 0.2.0). Updated `cran_comments.md` for CRAN submission. Also narrowed conditions for R/Python GitHub workflows.

### Checklist

N/A

### Additional Comments

N/A

### CODE REVIEW

File: R-package/revdep/email.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+release_date: ???
```
Comment (bgrnwd) [inline R-package/revdep/email.yml:1]: Do we need these filled in?

**Comment by bgrnwd** [comment]
Mind also bumping the Python version?

**Comment by tyler-richardett** [comment]
> Mind also bumping the Python version?

Yeah, should be there now. Missed it the first time with `git add`.

File: R-package/revdep/email.yml (hunk 1 above)
Comment (tyler-richardett) [reply R-package/revdep/email.yml:1]: I think only if you're mass emailing package maintainers with downstream dependencies? Which we don't have at this point. 

Also worth noting this `revdep` library is still in an experimental state. I was just using it to quickly see if we had any downstream dependencies yet.

**Review by bgrnwd — APPROVED** [review]
lgtm

**Pull request merged**